### PR TITLE
Tradheli-Add swashplate setup wiki

### PR DESCRIPTION
**Pull request by bnsgeyer**
This PR provides a wiki for traditional helicopter that details how to setup the swashplate with the new swashplate library that was incorporated in Copter 4.0.  @ChristopherOlson  and @R-Lefebvre  please review and provide comment.  I think it is 90% complete.  I would like some input on the graphics and whether I should put all of the swashplate graphics in one below the list or leave as it is.  I think there are some other details that I still need to pull over from the [discuss thread](https://discuss.ardupilot.org/t/four-servo-swashplate-support/37936)

### CODE REVIEW

**Comment by ChristopherOlson** [comment]
This all LGTM. Except it should maybe be noted that the "generic" swashplate type with servo positions cannot be used with H3-140 swashplate, and that the H3-140 swashplate type should be selected exclusively for these. Went thru this with a user that tried to get the servo position setup to work with it by setting them to -40, +40, 180 and then tried to apply a strange mix in the RC to get it to work, including reversing the collective in the RC (the swashplate moves downward on this type to increase collective pitch). Obviously this had less than desirable results in anything but acro flight mode where it had been "hacked" to work.

**Comment by bnsgeyer** [comment]
@ChristopherOlson thanks for the feedback. Do you think I should put the motor numbers on the graphics instead of servo number?

**Comment by ChristopherOlson** [comment]
@bnsgeyer I would say no? I think using the servo number is more intuitive so folks have no question as to which port to plug the servo into. Theoretically you can plug the servos into any port and just assign it the right Motor Number. But the fact that the Motor Numbers are strange values in the code and param setting tends to be a bit confusing. I think it's fine as-is.

**Comment by bnsgeyer** [comment]
@ChristopherOlson Check it over one last time.  I made some minor adjustments and added a few pictures.  One of the pictures is from the head setup video in which I did a freeze frame.  Just want to make sure you are ok with me using that in this wiki.

**Comment by ChristopherOlson** [comment]
@bnsgeyer LGTM, absolutely no problem at all with the snapshot.

**Comment by bnsgeyer** [comment]
@Hwurzburg  I am happy with Chris being the only one that reviewed it.  Please remove the WIP tag and put it through final review for merging.
Thanks,
Bill

File: copter/source/docs/traditional-helicopter-swashplate-setup.rst [hunk 1]
```diff
@@ -0,0 +1,152 @@
+.. _traditional-helicopter-swashplate-setup:
+
+===============================
+Traditional Helicopter – Swashplate Setup
+===============================
```
[the remark this answers is not in the document]
Comment (bnsgeyer) [reply copter/source/docs/traditional-helicopter-swashplate-setup.rst]: thanks.  I wasn't sure what the formatting rules were for this.

File: copter/source/docs/traditional-helicopter-swashplate-setup.rst [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+linkage in the case of a gas engine that you could have binding occur while doing the 
+swashplate set up. This is due to the H_SV_MAN feature provides manual servo settings 
+that drive the servos to the min and max values which includes the tail rotor servo 
+and the throttle servo due to throttle curve settings.
+
+Select Swashplate Type
+================
+
+Below are the swashplate type selections using the H_SWASH_TYPE parameter.  The diagrams
+shown list the servo 1, servo 2 or servo 3 for the three servo swashplate types.  These
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply copter/source/docs/traditional-helicopter-swashplate-setup.rst]: sorry, the links still not formatted properly...not your fault, the example above renders wrong on this page and hides two tick marks...I will pull down the PR and fix then all and PR back to your branch and you can cherry-pick or merge and re-squash....

**Comment by bnsgeyer** [comment]
@Hwurzburg  Should I use the link to the parameter in my titles too?

**Comment by Hwurzburg** [comment]
@bnsgeyer I personally dont do that , I think it looks weird...only in the body

:ref:`Link Text <a_named_link>`  #Links to "a_named_link". Displays the specified text
params are a special case of this....I hope it renders, if not, I will delete it

File: copter/source/docs/traditional-helicopter-swashplate-setup.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply copter/source/docs/traditional-helicopter-swashplate-setup.rst]: looks like you have them fixed now....I'll check and build tomorrow

**Comment by bnsgeyer** [comment]
@Hwurzburg I think I got everything.  I will squash commits tomorrow.  thanks for the help

**Comment by Hwurzburg** [comment]
I will pull it down tomorrow and fix the link formats...

**Comment by bnsgeyer** [comment]
@hwurzburg  i’m sorry if I didn’t get the syntax down correctly for the links. I just wanted to make you aware that I intentionally left the MotorsHeli_single out of the  displayed name for those parameters because it applies to both frame types.

**Comment by bnsgeyer** [comment]
@Hwurzburg I think i finally figured out the syntax.  I'm squashing now.

**Comment by Hwurzburg** [comment]
if you can wait a minute, I spent several hours fixing formats, etc. will send you a PR to your branch for you to review and merge if you want

**Comment by bnsgeyer** [comment]
@Hwurzburg  oops.  sorry, I didn't realize you were already working it.  You can force push your changes.  I only spent 5 minutes correcting that stuff.  Thanks for your help!

**Comment by Hwurzburg** [comment]
no problem...look it over...let me know if you want me to push to your branch...
just built your last force...still has a bunch of linking and formatting issues....I had to talk to Peter myself since some of the param names ,those with parens, made Sphinx barf....we finally figured how to get them to link...

**Comment by bnsgeyer** [comment]
Just force push what  you have.  Like I said, I didn't spend much time and it would be easier to look over and adjust what you did since it is formatted correctly.

**Comment by bnsgeyer** [comment]
It looks like there are a lot of changes to other files.  I'm assuming it is other PR's that have been merged since my last rebase?  or did you change other files align with my PR

**Comment by Hwurzburg** [comment]
Done....unless you can build the page locally in HTML using Sphinx, using "View file" here just doesn't do it justice...or exactly show how it will look...

**Comment by bnsgeyer** [comment]
@Hwurzburg  made some minor adjustments to wording but thought it looked great.  Sorry that I lost your authorship in the squash.   I thought it would keep it.  merge at will

**Comment by bnsgeyer** [comment]
@Hwurzburg Is it possible to get this merged or are you waiting on Randy to review.  I wanted to rebase my other branch to work on some new PRs

**Comment by Hwurzburg** [comment]
Randy said it was okay not to review, but was letting him look it over....but if there is a reason...

Merging

**Comment by bnsgeyer** [comment]
@hwurzburg we can either Skype or go to a chat room in mumble. Mumble might be easier. Will I need to have access to a computer to look at github?

**Comment by Hwurzburg** [comment]
no..how about now?

**Comment by bnsgeyer** [comment]
Sure. I am on mumble

**Pull request merged**